walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -32,6 +32,17 @@ private static boolean isAttended(Event event, MeetingRequest request) {
     return false; // Fallback for if no attendees are found that belong to both the event and the meeting request
   }
 
+  /**
+    * Returns the minutes in a day from a half hour slot
+    *
+    @param int halfHour The halfhour slot to be converted
+    @exception IllegalArgumentException thrown if slot > 0 or slot > 47.
+  */
+  private static int getTime(int halfHour) {
+    if(halfHour < 0 || halfHour > 47 ) throw new IllegalArgumentException("Time out of bounds.");
+    return halfHour * 30;
```

Comment:
Nit: I would suggest to put a new line between the return statement and the input checking to show that those are not logically connected components.

Reply:
Updated with commit 815052a